torch/_torch_docs.py

```diff
@@ -4942,6 +4970,34 @@ def merge_dicts(*dicts):
             [-0.0760, -3.6705,  2.4784]])
 """.format(**common_args, **tf32_notes))
 
+add_docstr(torch.hspmm,
+           r"""
+hspmm(mat1, mat2, *, out=None) -> Tensor
+
+Performs a matrix multiplication of a :ref:`sparse COO matrix
+<sparse-coo-docs>` :attr:`mat1` and a strided matrix :attr:`mat2`. The
+result is a (1 + 1)-dimensional :ref:`hybrid COO matrix
+<sparse-hybrid-coo-docs>`.
+
+Args:
+    mat1 (Tensor): the first sparse matrix to be multiplied
+    mat2 (Tensor): the second dense matrix to be multiplied
```

Comment:
In the one-sentence description this was "strided," but now it's "dense"

Reply:
Fixed to "strided".